test/functional/rpc_net.py

```
@@ -198,8 +198,8 @@ def test_getnetworkinfo(self):
             self.nodes[0].setnetworkactive(state=False)
         assert_equal(self.nodes[0].getnetworkinfo()['networkactive'], False)
         # Wait a bit for all sockets to close
-        self.wait_until(lambda: self.nodes[0].getnetworkinfo()['connections'] == 0, timeout=3)
-        self.wait_until(lambda: self.nodes[1].getnetworkinfo()['connections'] == 0, timeout=3)
+        for n in self.nodes:
+            self.wait_until(lambda: n.getnetworkinfo()['connections'] == 0, timeout=3)
```

Ruff (0.11.9)

Function definition does not bind loop variable `n`

(B023)